### DISCUSSION TITLE
Release Notes CovPass and CovPassCheck Version 1.17

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-android#182] [Announcements]
📢 **Release Notes CovPass and CovPassCheck Version 1.17** 📢  
Version 1.17 of the CovPass and CovPassCheck is planned to be released early february. It will include a variety of features and fixes that were requested by the community.

**Planned Main Features CovPass:**
•	Improvements on Trustlist-Updates: The apps will now use etag, gzip and other http-features to improve the update-mechanisms of the trustlist (Already present for IOS / New for Android).

**Planned Main Features CovPassCheck:**
•	The CovPassCheck now offers scan-sequences to check your 2G+ Status. The awaited booster-recognition, especially regarding the use-case “valid test not necessary after booster vaccination” is planned to be added with the next release 1.18.

**Planned Bug-Fixes:**
•	Multiple UI-Bugfixes as Accessibility-Enhancements are planned for the release.  
  
Thank you all for your feedback, ideas and critics!

stay safe and best regards,

Team CovPass and CovPassCheck ✔️

**Comment by herryilmaz** [comment]
When(if?) the issue for the people who had previously Corona will be solved by the App? I only had to be vaccinated once and received directly 2/2 for the number of the doses on my vaccination certificate. I also got my booster vaccination (technically second) but also on these it's written 2/2 so the app doesn't see this as a "Booster" vaccination, but rather just a second one.

I'm of course not the only one with this issue: https://www.br.de/nachrichten/netzwelt/app-chaos-genesene-muessen-sich-nicht-dreimal-impfen-lassen,SuCw9JI

Is it ever going to be solved?!

**Reply by Ein-Tim** [reply]
You should have gotten one 1/1 and one 2/2 cert.
Or, after the EU change yesterday, you should have a 1/1 and a 2/1 cert.

**Reply by herryilmaz** [reply]
As you can also read it from the linked news above, there is/was no standard with "numbering" the certificates. I know people who received 1/1 or directly 2/2 like me but regardless these will not solve the issue as when a recovered person gets a Booster shot, vaccination centers are still giving 2/2, not "3/3" as it's not the third one. So If I add a second certificate with again 2/2 the app should recognize this as a Booster. Or some millions of people need to head to vaccination centers again for fresh certificates. (and try to explain them how CovPass works :D )

**Reply by Ein-Tim** [reply]
A booster which is the second vaccination should be issued as 2/1.

**Reply by bobo-le** [reply]
@Ein-Tim As you mentioned i got a 2/1 cert yesterday. My booster-vaccination was on 25. january:
![grafik](https://user-images.githubusercontent.com/15114434/152496186-54a2c0da-2ede-4e8e-b6ac-34e213b888bb.png)

Now it's showing correctly in the covpass app:
![grafik](https://user-images.githubusercontent.com/15114434/152496345-896ee508-faba-45fe-9b4f-9b8849bfcf36.png)

But when I check the certificate with the check app, it still shows as invalid:
![grafik](https://user-images.githubusercontent.com/15114434/152496530-c6d31c22-13a0-4591-b740-9659c1d4a9e1.png)

**Reply by Ein-Tim** [reply]
@bobo-le

That's strange. I don't know what is going on there.

**Reply by timokoenig** [reply]
I would believe that CovPass takes the booster into account and show the right status but the EU rules (especially https://distribution.dcc-rules.de/rules/DE/f19e4792aeaffc8aba2ad59a851cbd3ceea6c0f44cafdb31cbeea6a6dcc7610e) have still not been updated. Therefore the validation is still wrong

**Reply by bobo-le** [reply]
@timokoenig thank you.
Do you know who updates these rules?

**Reply by timokoenig** [reply]
@bobo-le sadly no. I would guess part of it would be BMG or RKI but the CovPass Team should know

**Reply by dadaphl** [reply]
> @Ein-Tim As you mentioned i got a 2/1 cert yesterday. My booster-vaccination was on 25. january: 
> 
> Now it's showing correctly in the covpass app: 
> 
> But when I check the certificate with the check app, it still shows as invalid:

I have exactly the same issue. Very frustrating. Even though I got the certificate remade in a different pharmacy the issue still persists. Seems like the developers are overwhelmed with the complexity of the 3g/2g+ rules in the J&J edge case. Btw. the swiss app can validate my certificate as booseter (2g+).
